connectors/kafka/README.md

```diff
@@ -0,0 +1,21 @@
+# README #
+
+## Kafka Connector ##
+The kafka connector allows the forklift to utilize kafka and confluent's schema registry.  
+
+## JMS ##
+Forklift was developed against the JMS spec.  Kafka is not a JMS implementation so some adaptation
+has occurred in order to allow Kafka to function with Forklift.
+
+### Queues and Topics ###
+Kafka only provides one form of topic which is analogous to ActiveMQ's Virtual Topic.  Only one member from
+each subscribed group will receive a message.  Requests for a queue or topic will return the 
+same producer type.
+
+### At Most Once Delivery ###
```

Comment:
Minor: this section is a bit awkward and could use a little work.

This isn't that bad, but hard to understand documentation is sometimes worse than no documentation.